utils/config/nodes.ipp

```diff
@@ -382,9 +382,14 @@ config::base_set_node< ValueType >::push_lua(lutok::state& /* state */) const
 template< typename ValueType >
 void
 config::base_set_node< ValueType >::set_lua(
-    lutok::state& /* state */,
-    const int /* value_index */)
+    lutok::state& state,
```

Comment:
This change seems superfluous.

Reply:
What do you mean? It adds a capability to read a string set config values which are needed for the `execenv` engine config variable. It looks the function was left as a stub for future implementation when it's really needed. The patch adds string set support.

Reply:
You're right -- it's not really.